subsys/testsuite/ztest/include/ztest_test_new.h

```diff
@@ -210,6 +210,16 @@ static inline void unit_test_noop(void)
 #define Z_ZTEST(suite, fn, t_options) Z_TEST(suite, fn, t_options, 0)
 #define Z_ZTEST_F(suite, fn, t_options) Z_TEST(suite, fn, t_options, 1)
 
+/**
+ * @brief Skips the test if config is enabled
+ *
+ * Use this macro to in at the start of your test case, to skip it when
+ * config is enabledd.  Useful when your test is still under development.
+ *
+ * @param config The KConfig option used to skip the test.
```

Comment:
Kconfig or statically assertable statement.

This call also be something like `ARRAY_SIZE(my_array) == 0`